lazy_tensor_core/reuse_compiled_graph.py

```diff
@@ -0,0 +1,71 @@
+from lazy_tensor_core import _LAZYC
+_LAZYC._ltc_init_ts_backend()
+import torch
+from torch import nn
+import copy
+import dis
+import sys
+import inspect
+
+class ModuleConstScale(nn.Module):
+    def __init__(self):
+        super(ModuleConstScale, self).__init__()
+
+    def forward(self, a):
+        return a * 2
+
+class ModuleSub(nn.Module):
+    def __init__(self):
+        super(ModuleSub, self).__init__()
+
+    def forward(self, a, b):
+        return a - b
```

Comment:
I think to illustrate a harder case, you can try to use an op like this 

x.addcdiv_(a, b, value=c)

where c is a scalar const that lazytensor will wrap into a tensor, and I am afraid it will complicate your parameter matching scheme.

Reply:
Thanks Will! I also found some simple repro to trigger the case you mentioned in class ModuleConstScale .
`return a * 2`

if a is an lazytensor, 2 will be wrapped into a BackendData.

Reply:
> return a * 2
yea, I think this should trigger the same issue.  I just gave you the exact op i had trouble with before in case it was subtly different. 

Note: we specially handle constants like '0' and '1' so those may behave differently.